### PR TITLE
Moved pylint flags to .pylintrc

### LINKED ISSUE #136: Python: Move pylint flags to config
## Description
We currently add several flags to pylint in the CI's Python Checkstyle step.   Moving this configuration to a config file would make it easier to maintain over time.

## Current Behavior
[smoke-tests.yml](https://github.com/GoogleCloudPlatform/bank-of-anthos/blob/master/.github/workflows/smoke-tests.yml)
```
   - name: Python Checkstyle
      timeout-minutes: 20
      run: |
        pylint --disable=F0401 --disable=W0511 --disable=E1101 --disable=R0801 --disable=R0903  ./src/*/*.py
        # disabled warnings: import loading, todos, function members, duplicate code, public methods
```

## Desired Behavior
```
   - name: Python Checkstyle
      timeout-minutes: 20
      run: |
        pylint 
```

## Open Questions
- [ ] should this be by service or for the entire project?

### PR DESCRIPTION
**Pull request by rajatdiptabiswas**
Fixes #136

Changes made:
1. Generated a default `.pylintrc` file using `pylint --generate-rcfile > .pylintrc` in the root directory
2. Removed all the default disable warnings from the `.pylintrc` file and added our own disable warning IDs instead from the `smoke-tests.yml` file
3. Updated the line in `smoke-tests.yml` file to `pylint --rcfile=./.pylintrc ./src/*/*.py`

### CODE REVIEW

**Comment by rajatdiptabiswas** [comment]
@googlebot I signed it!

**Comment by bharathkkb** [comment]
@rajatdiptabiswas thanks for the PR
Like you mentioned in your comment [here](https://github.com/GoogleCloudPlatform/bank-of-anthos/issues/136#issuecomment-619332175), I think we can get rid of the defaults.

I will defer to @Daniel-Sanche for confirmation on this, but I think we will only need this in the file.
```
[MASTER]
[MESSAGES CONTROL]
disable=F0401,
        W0511,
        E1101,
        R0801,
        R0903,
        R0915

# disabled warnings: import loading, todos, function members, duplicate code,
# public methods, too many statements

```

File: .pylintrc [hunk 1]
```diff
@@ -0,0 +1,500 @@
...
+# Disable the message, report, category or checker with the given id(s). You
+# can either give multiple identifiers separated by comma (,) or put this
+# option multiple times (only on the command line, not in the configuration
+# file where it should appear only once). You can also use "--disable=all" to
+# disable everything first and then reenable specific checks. For example, if
+# you want to run only the similarities checker, you can use "--disable=all
+# --enable=similarities". If you want to run only the classes checker, but have
+# no Warning level messages displayed, use "--disable=all --enable=classes
+# --disable=W".
+disable=F0401,
```
Comment (bharathkkb) [inline .pylintrc:63]: small nit: in terms of readability I think having the comment about what the message code side by side with the code makes it easier.
```suggestion
disable=F0401, # import loading
```

Comment (rajatdiptabiswas) [reply .pylintrc:63]: Sure I can do that!

**Comment by Daniel-Sanche** [comment]
Yeah I agree with @bharathkkb, it would be great if we could cut out more of the boilerplate config

Thanks or contributing!

**Comment by rajatdiptabiswas** [comment]
Thanks for the clarification! @Daniel-Sanche @bharathkkb 

I shall make all the mentioned changes and create a new PR. I'm closing this one for now and I shall open a new one as soon as I'm done making the changes.

**Pull request closed without merging**